main.py

```
@@ -148,213 +91,234 @@ async def send_message(self, connection_id: str, message: str):
+async def serve_home():
+    return HTMLResponse(content="<h1>AI Proxy v3.0</h1><p>Visit /docs for API documentation or use the Next.js frontend.</p>")
 
 if __name__ == "__main__":
     import uvicorn
-    logger.info(f"Available CPUs: {os.cpu_count()}")
-    logger.info(f"Initial cache maxsize: {get_sync_initial_cache_maxsize()}")
-    logger.info(f"Initial v1 customer limit: {get_sync_dynamic_customer_limit_str()}")
-    logger.info(f"Initial v2 business limit: {get_sync_dynamic_business_limit_str()}")
-    logger.info(f"Initial default limit: {get_sync_dynamic_default_limit_str()}")
-    uvicorn.run("main:app", host="0.0.0.0", port=8100, reload=True, ws_ping_interval=30, ws_ping_timeout=60)
+    uvicorn.run("main:app", host="0.0.0.0", port=8100)
```

Ruff (0.15.14)

[error] Possible binding to all interfaces

(S104)